docs/dev_guide/contents/pr_review_procedure.rst

```diff
@@ -25,11 +25,19 @@ Before the "merge" button is clicked the following criteria must be met:
 
 * All the continuous integration must pass unless there is a known issue.
 
-* At least two members (not the author of the PR) of the "sunpy-developers" group must have approved the PR, one should, ideally, be a relevant subpackage maintainer.
+* At least two members (excluding the PR author) of the "sunpy-developers" group must have approved the PR.
+  One should, ideally, be a relevant subpackage maintainer.
 
 * All comments posted on the thread must be resolved.
 
-It is important that approval for merging the PR is done on the comment thread, as this becomes part of the "permanent record", this includes in during community meetings or in chat.
+It is important that approval for merging the PR is always done on the comment thread, as this becomes part of the "permanent record".
+
+Minor changes
+-------------
+
+If a PR only makes minor changes, it can be merged by the first reviewer, if they are confident they fully understand the changes.
+If this happens, the reviewer who merges should add the ``DocFix`` or ``CodeFix`` labels to the PR to indicate that they took a conscious decision that it only needed on reviewer.
```

Comment:
If I recall bugfix also has this one review is required note. Do we want to change that?

Reply:
I thought that being a bugfix didn't automatically imply the change being a trivial one that only needed one reviewer, so I'd be pro removing that tag (or re-purposing it to denote a bug fix PR).

Reply:
The "Bugfix" label has historically meant that it required only one reviewer

Reply:
Sorry, perhaps a bit of confusion with my language 😄 - what I meant was that in my opinion fixing a bug should not always imply that a PR only needs one reviewer.

Reply:
No disagreement there

Fixed sentence:
The "Bugfix" label has historically meant that the PR not only fixed a bug, but was also sufficiently simple and localized as to necessitate only one reviewer

Reply:
I've simplified this sentence now to recommend a single label, `minor-change`, that we can apply if a change is considered minor enough for one reviewer. Thoughts?